public/main.js

```diff
@@ -13,6 +15,17 @@ form.onsubmit = function(event) {
     event.preventDefault();
 };
 
+//when clicked all the currently completed todos are deleted
+deleteCompletedBtn.onclick = function() {
```

Comment:
If you rename this to deleteCompletedTodoBtn, you can remove the above comment as it will be unnecessary.